book/src/developers/quick_setup.md

```diff
@@ -196,12 +198,12 @@ cargo run -p trin-cli -- json-rpc discv5_nodeInfo
 
```

Comment:
We no longer maintain `trin-cli`. It doesn't work well once you start using multiple parameters. I'm not sure we should "advertise" using it and give these examples using `web3.py` instead. @ogenev thoughts?

Reply:
> We no longer maintain `trin-cli`. It doesn't work well once you start using multiple parameters. I'm not sure we should "advertise" using it and give these examples using `web3.py` instead. @ogenev thoughts?

I agree, I have never used `trin-cli` and I don't think we need to maintain it.

Reply:
Opened:
- https://github.com/ethereum/trin/issues/573

Reply:
I have removed all references to trin-cli from the book and relocated them to trin-cli to be addressed in the above issue.